.changeset/silent-dodos-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
```

Comment:
```suggestion
"@rocket.chat/meteor": minor
```

Reply:
we may want to have this as a `patch` since we'll probably release in older versions as well